docs/categorical-histogram-dap-04.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: /categorical-histogram-dap-04
+sidebar_position: 2
```

Comment:
Consider putting this in a separate "Product Documentation" category, since this topic is specific enough that it probably doesn't belong in the top level category (I'm open to better names for a category of "general topics of interest").

This will also let us just leave sidebar_position out and let it be alphabetically sorted.

Here's how those changes would look https://github.com/divviup/public-docs/commit/1e2ebe68638ec473c6a9bf01f0398314559e6f5b (I took out the slug since it isn't strictly necessary, but can keep it if you want)

Reply:
Thanks! I'm going to merge that commit in.